### ISSUE TITLE
ambiguous versioning schemes 

### ISSUE THREAD

**Issue by blshkv** [libyal/libvshadow#12]
Hi,

I have reported the issue in https://github.com/repology/libversion/issues/4 but it seems may something to do with the way how you release it. 

Here is the quote:
```
No, broken and ambigous upstream (and I mean both software projects and repositories here) versioning schemes cannot and will not be fixed in libversion.

Here 'alpha' it's not even part of the version. Tag is numeric, version in configure.ac is numeric, 'alpha' is only mentioned in release name and even there it preceeds version which suggests it's not a part of it. No other repo use this suffix either.
```

This is all confusing. Could you name releases (all libc* affected) to would match a version in a source code?

**Comment by joachimmetz** [comment]
alpha is a status indicator is not part of the version. The version is YYYYMMDD, whas is ambiguous about this?

https://en.wikipedia.org/wiki/Software_release_life_cycle

**Comment by blshkv** [comment]
The problem is that the "status indicator" must be part of the version, see:
https://en.wikipedia.org/wiki/Software_versioning 

as of now it is more like a random text which came up from nowhere in the release name. 
But even the name of directory inside of it is call differently (without alpha).

**Comment by blshkv** [comment]
Also, see:
https://www.gnu.org/prep/standards/html_node/Releases.html
```
Package the distribution of Foo version 69.96 up in a gzipped tar file with the name foo-69.96.tar.gz. It should unpack into a subdirectory named foo-69.96.
```

**Comment by joachimmetz** [comment]
> The problem is that the "status indicator" must be part of the version

this highly depends on what standard you adhere to.

neither sources you reference are authoritative for the visioning used in libyal

**Comment by blshkv** [comment]
Well. I tried my best. Not going to fight with a wind

**Comment by joachimmetz** [comment]
@blshkv in my point of view you did not try anything but enforce your opinion. 

IMHO as a maintainer you want to build up relationships with developer not "scream and shit all over them". (https://en.wikipedia.org/wiki/Seagull_manager)

**Comment by blshkv** [comment]
that wasn't an opinion. I described a problem which I was facing. I pointed to an another bug report. I linked to standards. You said "non authoritative" to everything. I dont time for this and this is your call really.

The ambiguous point is that all linux distros call it differently, see:
https://repology.org/metapackage/libvshadow/versions
Majority had to drop "alpha" keyword, some call it 0.version (because 0 can represent alpha).
I tried to call version_alpha but the package got flagged as "outdated", see that libversion bug 4 for details.

But I'm glad that you wrote that last comment. I will leave this bug open and let you make the final decision

**Comment by joachimmetz** [comment]
The TL;DR is that the packaging version is up to the distro. libyal will remain to use a very simple versioning schema YYYYMMDD. The status indicator can be ignored in the version.

> I described a problem which I was facing. I pointed to an another bug report. 

* You pointed to a bug report for which it is unclear how that relates to libyal.
* You claim the version is ambiguous but did not explain why
* You make statements about things **must**

I really recommend you to write more clear bug reports (explain and add context) and be less pushy in your communications. 

> The ambiguous point is that all linux distros call it differently

How distros do their packaging versioning is not the responsibility of libyal. Some use the .so number in the package name as well. All have different strong opinions any version change in libyal is not going to unify that.

**Comment by blshkv** [comment]
RIP
